README.md

```diff
@@ -1,39 +1,90 @@
 # Numeric Separators
 
-This is a [strawman proposal](https://tc39.github.io/process-document/) to introduce a separator in numeric literals in [ECMAScript](https://github.com/tc39/ecma262/).
+## Stage 1
+
+This is a [proposal](https://tc39.github.io/process-document/) to introduce a separator in numeric literals in [ECMAScript](https://github.com/tc39/ecma262/).
 
 ## Motivation
 
-This feature enables developers to make their numeric literals more readable by creating a visual separation between groups of digits.
+This feature enables developers to make their numeric literals more readable by creating a visual separation between groups of digits. 
+
+
+## Examples
+
+(The following examples also appear in the README.md of Babel transform plugin for this proposal.)
+
+
+### Decimal Literals
+
+```js
+let budget = 1_000_000_000_000;
```

Comment:
Maybe I misunderstood https://github.com/samuelgoto/proposal-numeric-separator/issues/4#issuecomment-304384535 - does "to a single character" mean "only one separator per number", or "each separator can only be a single code unit between digits"?